common/persistence/task_manager.go

```diff
@@ -235,13 +235,6 @@ func (m *taskManagerImpl) GetTasks(
 	return &GetTasksResponse{Tasks: tasks, NextPageToken: internalResp.NextPageToken}, nil
 }
 
-func (m *taskManagerImpl) CompleteTask(
-	ctx context.Context,
-	request *CompleteTaskRequest,
-) error {
-	return m.taskStore.CompleteTask(ctx, request)
```

Comment:
Remove this from each taskStore also?

Reply:
As in? Think I have removed it for now since I had a look and it felt like it wasn't being used in a lot of places (except for one single unit test)